src/python/aqueduct_executor/operators/airflow/dag.template

```diff
@@ -24,6 +24,28 @@ VENV_REQUIREMENTS=[
 ]
 
 def invoke_task(spec, **kwargs):
+    '''
+    Install specific version if version_tag is provided.
+    '''
+    if "{{ version_tag }}":
+        import subprocess
+        import sys
+
+        install_process = subprocess.run([
+            sys.executable,
+            "-m",
+            "pip",
+            "install",
+            "--index-url",
+            "https://test.pypi.org/simple/",
+            "--extra-index-url", # allows dependencies from pypi
+            "https://pypi.org/simple",
+            f"aqueduct-ml=={{ version_tag }}",
```

Comment:
Please change this to `"aqueduct-ml=={{ version_tag }}"` The prefix of `f` is not necessary since this is a jinja template. Found this during my testing,

Reply:
Thanks! Updated